libgen_client.py

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env python3
+import getopt
```

Comment:
Could you move this import into `main` so the plugin still works if `getopt` isn't installed?